apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModalTitle.tsx

```diff
@@ -61,7 +61,7 @@ const PolicyEditorModalTitle = ({
             Try Supabase Assistant
           </Button>
         )}
-        <Button asChild type="default" icon={<IconExternalLink size={14} />}>
+        <Button asChild type="default" icon={<IconExternalLink size={14} />} className='mt-[-4px]' >
```

Comment:
```suggestion
        <Button asChild type="default" icon={<IconExternalLink size={14} />} className='mt-[-4px]'>
```

Reply:
Prettier is failing, removing that final space should fix it @Mohammed-Yasin-Mulla

Reply:
thank you

Reply:
@fabianmossberg, did the refactoring of the code